sdk/schemaregistry/schema-registry/src/mappers.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createSerializer, OperationSpec } from "@azure/core-client";
+import * as Mappers from "./generated/models/mappers";
+import * as Parameters from "./generated/models/parameters";
+
+const serializer = createSerializer(Mappers, /* isXml */ false);
```

Comment:
might be worth doing this in swagger instead of by hand.

Reply:
Done!